Venter/models.py

```diff
@@ -50,6 +66,13 @@ def __str__(self):
 
 
 class Header(models.Model):
+    """
+    A Header list associated with each organisation.
+    Eg: Organisation xyz may contain headers in the csv file such as- user_id, title etc
+
+    # Create a header instance
```

Comment:
In the previous comments I have said that you don't need to write sample code here. The main reason for that is that any new intern is going to be expected to look at the django documentation, and solve the assignment before they're going to be allowed to look at the code, Thus, at the very least, we can expect them to understand the code we have written as long as we explain what it is (i.e using a one line description in the docstring for what the model class is, and what it does). Thus, it's redundant to write "Header creation" code, since its a standard instance of Djanog functionality.

**_What you did here would be warranted if we were using some trickery or a non-standard, slightly awkward way of doing things, and elaborating our special use case through a sample code snippet would better elucidate our point. This is not one of those cases._**

Reply:
@samjakes , I see your point, but let's consider moving the sample code in the comments during the final review and code-cleaning phase?